data/json/items/chemicals_and_resources.json

```diff
@@ -540,31 +540,73 @@
     "count" : 10
    },
    {
-    "type" : "AMMO",
+    "type" : "COMESTIBLE",
     "id" : "chem_ethanol",
-    "category" : "spare_parts",
-    "price" : 100,
+    "comestible_type" : "DRINK",
+    "use_action" : "ALCOHOL_STRONG",
+    "nutrition" : 0,
+    "quench" : -16,
+    "healthy" : -10,
+    "addiction_type" : "alcohol",
+    "addiction_potential" : 7,
+    "spoils_in" : 0,
+    "flags" : ["EATEN_COLD"],
+    "fun" : 2,
+    "charges": 17,
```

Comment:
Why 17 and not 14 like alcohols? It breaks crafting proportions. You could bump it to 21 to keep the multiple of 7 thing we have going on.